java/src/main/java/actorworkflow/CustomerLoyaltyActivitiesImpl.java

```diff
@@ -0,0 +1,17 @@
+package actorworkflow;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CustomerLoyaltyActivitiesImpl implements CustomerLoyaltyActivities {
+    private final boolean HAS_BUG = false;
```

Comment:
not used?

Reply:
Oops. Carry over from an old manual test. Removing.